clients/admin-ui/src/features/common/RequestType.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Tag } from '@fidesui/react';
+import React from 'react';
+
+import { ActionType, Rule } from '../privacy-requests/types';
+
+type RequestTypeProps = {
+  rules: Rule[];
+};
+
+const RequestType = ({ rules }: RequestTypeProps) => {
+  const actions = Array.from(
+    new Set(
+      rules
+        .filter((d) => Object.values(ActionType).includes(d.action_type))
```

Comment:
We're going to display the request type using the action type on rules now. This component handles de-duping the rules just in case there are multiple rules that are the same type.